rasa/server.py

```diff
@@ -238,9 +239,22 @@ def _create_emulator(mode: Optional[Text]) -> NoEmulator:
     model_path: Optional[Text] = None,
     model_server: Optional[EndpointConfig] = None,
     remote_storage: Optional[Text] = None,
+    endpoints: Optional[AvailableEndpoints] = None,
 ) -> Agent:
     try:
-        loaded_agent = await load_agent(model_path, model_server, remote_storage)
+        _broker = broker.from_endpoint_config(endpoints.event_broker)
```

Comment:
given that `endpoints` is optional, it can be `None` (and is by default). We need to handle this, otherwise `endpoints.event_broker` will throw an exception.

Reply:
fixed